pygmt/figure.py

```diff
@@ -235,62 +255,79 @@ def savefig(
         if show:
             launch_external_viewer(fname)
 
-    def show(self, dpi=300, width=500, method="static"):
+    def show(self, method=None, dpi=None, width=None):
```

Comment:
Maybe I should clarify what I meant at https://github.com/GenericMappingTools/pygmt/pull/529#discussion_r578027757. I think that this PR should only change the `method` part to `method=None` (auto detect) and keep the `dpi` and `width` settings untouched.

```suggestion
    def show(self, dpi=300, width=500, method=None):
```

Reply:
I just reverted the changes about *dpi* and *width*.